Automattic/wp-calypso
# Move product logos to components package

Related to #76442

## Proposed Changes

* Adds logos added for the new Plans2023 grid to Automattic packages and adds a demo of said logos.

![image](https://github.com/Automattic/wp-calypso/assets/3422709/6648e6c8-fcde-4a04-84f3-8a2c27affd8c)

Caveat: There seem to be several other logos scattered throughout the codebase, however they are not scoped into this change. The main motivation for this change is to decouple the plan comparison grid so that it can be moved into a package as laid out in #76442

## Testing Instructions
- There shouldn't' be any visible changes from production, except for the new Logo section.
-  Go through the plans step `/start/plans` of the sign up flow and make sure everything works in the plans step and the all logos are displayed properly.
- Go to the `/plans` page and make sure of the same


*

## Pre-merge Checklist
